README.md

```diff
@@ -5,4 +5,8 @@ A lambda that polls RCS for rights updates, parses the returned xml and sends it
 ![Architecture Diagram](rcs-poller-architecture.png?raw=true)
 
 ### Run locally
-Get `composer` credentials and run: `sbt run`
\ No newline at end of file
+Get `composer` credentials and run: `sbt run`
+
+### Deploy
+
+Deploy this project using riffraff. You can push your branch to the repository, test the changes out in CODE, then deploy to PROD upon merging your pull request to master.
```

Comment:
Should we setup continuous deployment for master and PROD? This will make getting a change on to PROD easier as it's just a merge.

Reply:
Absolutely, could we pair on that? Would be great to know how.

Reply:
Yep, happy to pair! RiffRaff has a list of CD projects here https://riffraff.gutools.co.uk/deployment/continuous.